test/controllers/api/v2/auth_sources_controller_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class Api::V2::AuthSourcesControllerTest < ActionController::TestCase
+  test "should get index" do
+    get :index, { }
+    assert_response :success
+    auth_sources = ActiveSupport::JSON.decode(@response.body)
+    assert !auth_sources.empty?
```

Comment:
I think it would be good to verify we get various auth source types, e.g. by something like

```
assert_includes ids, auth_sources(:internal).id
```